packages/remix-dev/compiler/compileBrowser.ts

```diff
@@ -135,7 +135,7 @@ const createEsbuildConfig = (
       "react/jsx-runtime",
       "react-dom",
       "react-dom/client",
-      "react-refresh/runtime",
+      require.resolve("react-refresh/runtime"),
```

Comment:
other packages listed are installed in the user's app already, so we shouldn't need to resolve them

Reply:
did it anyway to be consistent and safe